Bukkit/src/main/java/com/plotsquared/bukkit/listener/PlayerEventListener.java

```diff
@@ -376,7 +376,7 @@ public void onEffect(@NonNull EntityPotionEffectEvent event) {
 
     @EventHandler
     public void onVehicleEntityCollision(VehicleEntityCollisionEvent e) {
-        if (e.getVehicle().getType() == EntityType.BOAT) {
+        if (e.getEntity() instanceof Boat) {
```

Comment:
That looks odd, I assume this should still be `e.getVehicle()`?

Reply:
yup, mb